2022/incoming/cgi-this.pod

```diff
@@ -0,0 +1,55 @@
+Author: Julien Fiegehenn <simbabque@cpan.org>
+Title: northpole.cgi
+Topic: App::CGIThis
+
+=encoding utf8
+
+=head1 northpole.cgi
+
+As is the case in most public sector institutions, a lot of the tools at the North pole are out of date.
+The elves often need to work with terrible spaghetti CGI scripts that are still around from the olden days.
+Quite a few of their internal tools with web interfaces were originally written using L<CGI>, and despite the
```

Comment:
The term "CGI script" is used quite a bit in this article so at the end I took a moment to refresh my memory on specifically what that was (e.g. as differentiated from the other kinds of Perl-based web things I work on now) as it's been quite a long time since my eyes have rested upon one. I've come to the understanding that a "CGI script" is one that uses CGI.pm. This line implies that, but for whatever reason it wasn't immediately obvious to me. I think most of your audience will perhaps know that more instinctively than I did. This comment is _here_, however, as I have a suggestion for this line that I think would have made that more obvious to me without needing to seek additional clarification.

Instead of "were originally written", I'm thinking "are still implemented" would make it more obvious to the reader (at least me) that their (my) CGI.pm-based scripts are the things App::CGIThis can work with.

Reply:
That is a very good suggestion, thank you!